env.sample

```diff
@@ -27,10 +29,24 @@ export PH_OAUTH_CLIENT_ID=opsapp
 
 export PH_OAUTH_CLIENT_SECRET=Y2xpZW50Og=
 
+export PH_OAUTH_CLIENT_UUUID=Y2xpZW50Og=
+
 export PH_OAUTH_BASIC_AUTH=true
 
 export PH_OAUTH_BASIC_AUTH_TOKEN=Y2xpZW50Og==
 
+export PH_JBPM_API_URL=http://localhost:8180/kie-server/services/rest/server
+
+export PH_JBPM_CONTAINER_ID=PaymentHubEE_1.0.0-SNAPSHOT
+
+export PH_JBPM_CREDENTIALS_ADMIN_MAKER=Y2xpZW50Og==
```

Comment:
Could you please add instructions to create the secrets on K8s?

Reply:
These are environment variables required for using jbpm rest api services